imports/api/Constants.js

```diff
@@ -43,7 +46,21 @@ export const Constants = {
 	SELF_DRAW: "selfdraw",
 	NO_WIN: "nowin",
 	RESTART: "restart",
-	MISTAKE: "fuckup"
+	MISTAKE: "fuckup",
+
+	EAST: "east",
+	SOUTH: "south",
+	WEST: "west",
+	NORTH: "north",
+
+	PRIORITY: {
+		east: 3,
+		south: 2,
+		north: 1,
+		west: 0
+	}
 };
 
+Constants.WINDS = [Constants.EAST, Constants.SOUTH, Constants.WEST, Constants.NORTH];
```

Comment:
Those values don't exist until the Constants object is defined. I added this line afterwards so I didn't have to be redundant.